### PR TITLE
Amah/update e2e sample

### PR DESCRIPTION
**Pull request by AliMahmoudzadeh**
# Description
updated the sample according to the latest changes in the sdk.

# Checklist

- [ x ] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ x ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by balapv — APPROVED** [review]
Looks good to me

File: tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb [hunk 1]
```diff
@@ -1168,6 +1182,21 @@
     "> Expect this deployment to take approximately 6 to 8 minutes."
    ]
   },
+  {
```
Comment (sdgilley) [inline tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb:1185]: This cell contains subscription information, should not be published.

**Review by sdgilley — CHANGES_REQUESTED** [review]
Should not publish your subscription info.

File: tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb (hunk 1 above)
Comment (AliMahmoudzadeh) [reply tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb]: removed the cell

**Review by sdgilley — APPROVED** [review]

**Pull request merged**